torch/onnx/symbolic_opset8.py

```diff
@@ -129,7 +128,7 @@ def _try_cast_integer_to_float(g, *args):
     if arg0_type is not None:
         old_type = arg0_type
         if old_type not in floating_scalar_types:
-            args = tuple(_cast_Float(g, arg, False) for arg in args)
+            args = tuple(sym_opset9._cast_Float(g, arg, False) for arg in args)  # type: ignore[attr-defined]
```

Comment:
why does mypy not like this?

Reply:
good question. Looks like it may be a transitive import. I will clean it up

Reply:
Turns out `_cast_Float` is generated programmatically. I added comments and a todo to clean it up.